Automattic/wp-calypso
# Remove Auto-renew and renew buttons from purchases page

Related to # 1201210512993648-as-1204413355670312/f

## Proposed Changes
For Akismet free we want to disable the auto-renew button for users. There are a number of ways to done this, but simple check seems the most straight forward and safe. However, if there are any other options, please let me know.

For example, the product might be a one time purchase, but it will involve updating product tables and deal with ones already activated. Also, `canAutoRenewBeTurnedOff` doesn't apply on all buttons to be removed, etc. 

## Testing Instructions
- checkout the branch locally and start it `yarn start`
- Purchase a Free Personal plan from `akismet.com`
- Navigate to `http://calypso.localhost:3000/me/purchases` and select akismet free subscription.
- Observe that there are no buttons to Renew Now and to disable auto-renew

<img width="977" alt="233198716-65888c94-8577-401c-a521-46046be06e58" src="https://user-images.githubusercontent.com/5714212/233391353-439ee0a0-9c6f-4883-87ed-865bbb15d36f.png">

## Pre-merge Checklist
